### ISSUE TITLE
aot-eager accuracy error: twins_pcpvt_base

### ISSUE THREAD

**Issue by desertfire** [pytorch/torchdynamo#1296]
Repro:
```
 benchmarks/timm_models.py --training -d cuda --use-eval-mode  --amp  --accuracy --backend aot_eager --only twins_pcpvt_base
```
Error:
```
torchdynamo.utils: [ERROR] RMSE (res-fp64): 1572.99442, (ref-fp64): 41.00998
FAIL
```

**Comment by eellison** [comment]
Fixed as of https://github.com/pytorch/pytorch/pull/85417 I believe
